utils/array.js

```diff
@@ -18,6 +18,21 @@ function chunks(array, size = 1) {
   return result;
 }
 
+/**
+ * Checks if two arrays have any common items
+ * @param array1 {Array} - first array
```

Comment:
Add Array<item> type into the description, as this function will be reused.
Also add this method is for singular values only such as strings and numbers.